contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-ios.mdx

```diff
@@ -30,17 +30,41 @@ To handle this, you can use the `didReceiveFeatureFlags` notification to wait fo
 
 ```swift
 // Before SDK initialization
```

Comment:
I think this comment should be moved when registering the notification because its related to that